### PR TITLE
added replace in go.mod

### PR DESCRIPTION
**Pull request by kunalgoyal9**
replace github.com/libp2p/go-libp2p-core => github.com/libp2p/go-libp2p-core v0.14.0

### CODE REVIEW

File: embedded_hub.go [hunk 1]
```diff
@@ -22,11 +17,17 @@ import (
...
+	"github.com/juju/loggo"
+	"github.com/libp2p/go-libp2p-core/crypto"
+	"github.com/libp2p/go-libp2p-core/peer"
+	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
 var localPeer *peer_service.PeerService
 
-var log = loggo.GetLogger("embedded_hub")
+// var log = loggo.GetLogger("embedded_hub")
```
Comment (sgladkov) [inline embedded_hub.go:30]: Why is this line commented?

File: go.mod [hunk 2]
```diff
@@ -1,23 +1,127 @@
-module lachain-communication-hub
+module main
```
Comment (sgladkov) [inline go.mod:1]: Don't sure we need main module here

File: host/builder.go [hunk 3]
```diff
@@ -106,9 +104,8 @@ func BuildNamedHost(priv_key crypto.PrivKey) core.Host {
 		listenAddrs = libp2p.ListenAddrs()
 	}
 	host, err := libp2p.New(
-		context.Background(),
 		listenAddrs,
-		libp2p.EnableRelay(circuit.OptHop),
+		libp2p.EnableRelay(),
```
Comment (sgladkov) [inline host/builder.go:108]: Why do we remove this option?

File: embedded_hub.go (hunk 1 above)
Comment (kunalgoyal9) [reply embedded_hub.go:30]: image
I was getting this error. So I commented it

Comment (sgladkov) [reply embedded_hub.go:30]: You should not compile main and embedded_hub simultaneously,  this are different modules, Don't use main module and don't add it to build please

File: host/builder.go (hunk 3 above)
Comment (kunalgoyal9) [reply host/builder.go:108]: I have removed this to sync with the latest interface of `libp2p.EnableRelay()`

File: go.mod (hunk 2 above)
Comment (kunalgoyal9) [reply go.mod]: fixed with lachain-communication-hub

File: embedded_hub.go (hunk 1 above)
Comment (kunalgoyal9) [reply embedded_hub.go]: got it.

**Review by sgladkov — APPROVED** [review]

**Pull request merged**